src/components/CriticalError.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { ErrorBoundary as ReactErrorBoundary } from 'react-error-boundary';
+import LogoWhite from '../assets/logo-sad-white.svg';
+
+import * as Styled from './CriticalError.styles';
+
+export function ErrorPage() {
+    return (
+        <Styled.Container role='alert'>
+            <Styled.Logo>
+                <LogoWhite />
+            </Styled.Logo>
+            <Styled.Message>
+                Whoops looks like something went wrong
+                <br />
+                Please try again
+            </Styled.Message>
+        </Styled.Container>
+    );
+}
+
+interface ErrorBoundaryProps {
+    children: React.ReactNode;
+}
+
+export function ErrorBoundary({ children }: ErrorBoundaryProps) {
+    return (
+        <ReactErrorBoundary
+            FallbackComponent={ErrorPage}
+            onReset={() => {
+                // reset the state of your app so the error doesn't happen again
```

Comment:
Is there an action item here? if so lets add a jira ticket

Reply:
may for a 'Try Again' button ?

Reply:
For error 500 it's not a screen, just a message and we should update the behaviour so legitimate errors won't be 500, and error 500 (internal server error) will take the user to the new vew.

Anyway, I can't technically use the existing views since the ErrorBoundary functions when there's a fatal **render** crash, so it has to be a different render code

Reply:
I removed the `onReset` for now, I wanted to have it so once we'll update the search behaviour, it'll reset the user back to that page.